.github/workflows/test.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# fail on non-zero return code from a subprocess
+set -e
+
+# download NC test loaction if the test needs the data and run tests
+if [ $1 == "NC" ]
+then
+  g.extension g.download.location
+  g.download.location url=https://grass.osgeo.org/sampledata/north_carolina/nc_spm_full_v2alpha2.tar.gz path=/grassdb
+  g.mapset mapset=PERMANENT location=nc_spm_full_v2alpha2 -c
```

Comment:
```suggestion
  g.download.location url=https://grass.osgeo.org/sampledata/north_carolina/nc_spm_08_micro.zip path=/grassdb
  g.mapset mapset=PERMANENT location=nc_spm_08_micro -c
```

Would the "micro" location (50 MB) do as well? The "full" location is tree times larger...

Reply:
Currently, the location is not downloaded at all, but an empty location is used (see Dockerfile). But maybe it's still useful for other testcases, that use the same test.sh, to change it? @anikaweinmann @mmacata